storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
> I don't see a value in providing noisy alternatives to using the properties.

I disagree, but not by much. I don't see **much** value, but I don't think it'd be noisy for users, it'd just be noisy for us to have to document.

But, let's end this discussion and just merge as-is?